mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -467,6 +468,14 @@ is set to true. Defaults to ZONAL.`,
 										AtLeastOneOf: insightsConfigKeys,
 										Description:  `True if Query Insights will record client address when enabled.`,
 									},
+									"query_plans_per_minute": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										Default:      5,
```

Comment:
Adding a new default value would make this a breaking change to existing configs. Instead of a default, could we change this line to `Computed: true` to let the API handle the default?

Reply:
done.